libraries/botbuilder-lg/src/LGFileParser.g4

```diff
@@ -38,6 +38,7 @@ parameters
 templateBody
 	: normalTemplateBody						#normalBody
 	| conditionalTemplateBody					#conditionalBody
+    | switchCaseTemplateBody                    #switchCaseBody
```

Comment:
let's either change conditionalTemplateBody to ifElseTemplateBody to make sure we clarify at this moment, or
 we merge switchCaseTemplateBody into conditionalTemplateBody, because it's all conditional.

Reply:
And the line is not aligned.